lib/shared/image_viewer.dart

```diff
@@ -113,6 +113,26 @@ class _ImageViewerState extends State<ImageViewer> with TickerProviderStateMixin
     );
   }
 
+  Future<void> getImageSize(String imageUrl, context) async {
+    final ByteData data = await NetworkAssetBundle(Uri.parse(imageUrl)).load(imageUrl);
+    final Uint8List bytes = data.buffer.asUint8List();
+    final decodedImage = await decodeImageFromList(bytes);
+
+    setState(() {
+      imagewidth = decodedImage.width;
+      imageheight = decodedImage.height;
+      maxzoomlevel = max(imagewidth, imageheight) / 128;
+      // ignore: avoid_print
+      print("$imagewidth + $imageheight + $maxzoomlevel");
```

Comment:
Ignoring a warning is a red flag. 😉 Use `debugPrint` instead of `print` for production.

https://dart.dev/tools/linter-rules/avoid_print

Reply:
done @micahmo